opentelemetry-otlp/Cargo.toml

```diff
@@ -51,6 +51,7 @@ opentelemetry_sdk = { features = ["trace", "rt-tokio", "testing"], path = "../op
 tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
 futures-util = { workspace = true }
 temp-env = { workspace = true }
+tonic = { workspace = true, features = ["server"] }
```

Comment:
interesting... is there anything in otlp test/example that requires server features?

Reply:
You can check this run - https://github.com/open-telemetry/opentelemetry-rust/actions/runs/13329777072/job/37230929028

Reply:
Got it, we have some smoke tests! I missed that.